File: backend/member/models/member.py
from django.db import models


class Member(models.Model):
    id = models.AutoField(primary_key=True)
    first_name = models.CharField(max_length=30)
    last_name = models.CharField(max_length=30)
    phone_number = models.CharField(max_length=30)

    Male = 'M'
    Female = 'F'
    sex_choices = [
        (Male, Male),
        (Female, Female),
    ]
    sex = models.CharField(max_length=30, choices=sex_choices, default=Male,)

    def __str__(self):
        return self.first_name + " " + self.last_name

File: frontend/src/app/models/member.ts
export class Member {
    public id?: string;
    public first_name?: string;
    public last_name?: string;
    public sex?: string;
    public phone_number?: string;

    public constructor(id: string, first_name: string, last_name: string, phone_number: string,sex: string) {
        this.id = id;
        this.first_name = first_name;
        this.last_name = last_name;
        this.phone_number = phone_number;
        this.sex = sex;
    }
}